### PR TITLE
fix: add encryptedAPKAMSymmetricKey to enrollment notification

### PR DESCRIPTION
**Pull request by murali-shris**
**- What I did**
- in enrollment notification encryptedApkamSymmetricKey is sent in json. encryptedAPKAMSymmetricKey is used in other places in server code. Temporarily send both encryptedApkamSymmetricKey,encryptedAPKAMSymmetricKey in the notification. Once server is released to prod, use encryptedAPKAMSymmetricKey for consistency
**- How I did it**
- added encryptedAPKAMSymmetricKey in enrollment notification json
- added a check in functional test
**- How to verify it**
- tests should pass

### CODE REVIEW

**Review by gkc — CHANGES_REQUESTED** [review]
@murali-shris please add assertions for the json payload in the notification to the tests in at_secondary_server/tests/monitor_verb_test.dart "Test delivery of enrollment request notification to PKAM" and "Test delivery of enrollment request notification to APKAM"

**Comment by murali-shris** [comment]
> @murali-shris please add assertions for the json payload in the notification to the tests in at_secondary_server/tests/monitor_verb_test.dart "Test delivery of enrollment request notification to PKAM" and "Test delivery of enrollment request notification to APKAM"

Added the checks for other notification values

File: packages/at_secondary_server/test/monitor_verb_test.dart [hunk 1]
```diff
@@ -360,6 +361,7 @@ void main() {
           ',"deviceName":"$deviceName"'
           ',"namespaces":${jsonEncode(namespaces)}'
           ',"apkamPublicKey":"dummy_apkam_public_key"'
+          ',"encryptedAPKAMSymmetricKey":"dummy_encrypted_apkam_symm_key"'
```
Comment (gkc) [inline packages/at_secondary_server/test/monitor_verb_test.dart:364]: @murali-shris Looks like we need to tighten up validation of enrollment requests to ensure that all required data is present. Suggest open another ticket and do another PR for that

**Review by gkc — APPROVED** [review]
LGTM
